src/components/NewsList.jsx

```diff
@@ -0,0 +1,47 @@
+import styles from "@/styles/components/NewsList.module.scss";
+import { useEffect, useState } from "react";
+import { fetchTopNews } from "@/api/api.js";
+import { useRecoilState, useRecoilValue } from "recoil";
+import {
+  isDayState,
+  newsState,
+  selectedCategoryState,
+} from "@/store/rootAtoms";
+import IntroMessage from "./IntroMessage";
+
+const NewsList = () => {
+  const [news, setNews] = useRecoilState(newsState);
+  const category = useRecoilValue(selectedCategoryState);
+  const isDay = useRecoilValue(isDayState);
+
+  useEffect(() => {
+    fetchTopNews(category).then(setNews);
+  }, [category]);
+
+  return (
+    <>
+      <div className={`${styles.newsList} ${!isDay ? styles.nightList : ""}`}>
+        <IntroMessage></IntroMessage>
+        {news.map((news, index) => (
+          <div
+            className={styles.newsCard}
+            key={index}
+            onClick={() => window.open(news.url, "_blank")}
+          >
+            <img
+              className={styles.newsThumbnail}
+              src={news.urlToImage}
+              alt="썸네일"
+            ></img>
```

Comment:
`<img>`태그는 셀프 클로징 태그로 뒤에 닫는 태그는 불필요하답니다!
```suggestion
            <img
              className={styles.newsThumbnail}
              src={news.urlToImage}
              alt="썸네일"
            />
```